dispatchrouter/pom.xml

```diff
@@ -94,7 +94,7 @@
               <images>
                 <image>
                   <build>
-                    <from>eclipsehono/qpid-dispatch:${qpid-dispatch.version}</from>
+                    <from>enmasseproject/qdrouterd:0.10.0</from>
```

Comment:
So, based on our discussion on IRC, this will later be changed to `enmasseproject/qdrouterd-base:0.8.0-1`, right?